.travis.yml

```diff
@@ -1,6 +1,9 @@
 language: go
 go:
-  - 1.7
+- 1.8
+install:
+- mkdir -p $HOME/gopath/src/k8s.io
```

Comment:
@brancz Maybe this is fine for yaml validation. I suggest we should have one or more leading space before the `-`(dash). This makes it more readable which is consistent with the goal of yaml. :)